releasing/versioning_pre_nightly.md

```diff
@@ -0,0 +1,56 @@
+## Versioning in nightly and prerelease binaries
+
+Binary packages produced for prerelease and nightly builds has some
+particularities to stablish the priority among them nicely.
+
+### Version schemes
+
+**Prerelease** versioning scheme: `{upcoming_version}~pre{prerelease_version}`
+
+ * `upcoming_version:` upstream version target for current prerelease series
```

Comment:
Does this follow the semantic versioning scheme? If so, it might be helpful to this example
```suggestion
 * `upcoming_version:` upstream version target for current prerelease series (e.g., `X.Y.Z`)
```

Reply:
indeed, all version schemes follow the semantic scheme. Thanks for the suggestion.